generic/tests/pktgen_perf.py

```diff
@@ -93,6 +94,29 @@ def _pin_vm_threads(node):
     for record in record_list.split():
         record_line += "%s|" % format_result(record)
 
+    # check module pktgen, install kernel-modules-internal package
```

Comment:
Please move this comment to the doc string of the function.

Reply:
ok ,updated.